### ISSUE TITLE
lit on 1.7.0

### ISSUE THREAD

**Issue by AnimaI** [raspiblitz/raspiblitz#2864]
I cannot install the LIT tool on my raspiblitz 1.7.0.
It does not install via the menu.
When I install the Manuel installation, the following message appears.

```
admin@raspberrypi:~ $ /home/admin/config.scripts/bonus.lit.sh on
making sure the lit service is not running
# INSTALL LIGHTNING TERMINAL
adduser: The group `lit' already exists.
admin@raspberrypi:~ $ 
```
How can I install LIT?

**Comment by openoms** [comment]
run:
`sudo groupdel lit`

**Comment by openoms** [comment]
closing as solved in v1.7.1
